latex/chain.tex

```diff
@@ -198,7 +198,7 @@ \subsection{Block Definitions}
   \begin{equation*}
     \begin{array}{r@{~\in~}lr}
       \fun{bheader} & \Block \to \BHeader \\
-      \fun{bhbody} & \Block \to \BHBody \\
+      \fun{bhbody} & \BHeader \to \BHBody \\
```

Comment:
another good type error catch! In figure 56 we apply `bhbody` to a block header. _but_, in figure `58` and in figure `62` we apply it to a block. So we just need to be consistent.